playwright/tests/fixtures/baseTest.ts

```diff
@@ -11,44 +11,22 @@ export const runInBrowser = process.env.RUN_IN_BRWOSER ? true : false;
 export const test = base.extend<{ workspace: string; page: Page }>({
   workspace: prebuiltWorkspacePath,
   page: async ({ workspace }, take) => {
-    const { page, browser, electronApp } = await appWithPage(workspace);
-    await initialize(page);
-    await take(page);
-    if (browser) {
-      await browser.close();
-      return;
-    }
-    if (electronApp) {
-      if (test.info().status === 'failed') {
-        const tracePath = test.info().outputPath('trace.zip');
-        const screenshotPath = test.info().outputPath('screenshot.png');
-        await page.context().tracing.stop({ path: tracePath });
-        await page.screenshot({ path: screenshotPath });
-        test.info().attachments.push({ name: 'trace', path: tracePath, contentType: 'application/zip' });
-        test.info().attachments.push({ name: 'screenshot', path: tracePath, contentType: 'image/png' });
-      }
-      await electronApp.close();
-    }
+    if (runInBrowser) await runBrowserTest(workspace, take);
+    else await runElectronAppTest(workspace, take);
```

Comment:
I know this is valid syntax, however I'm more used to read the brackets syntax...
```suggestion
    if (runInBrowser) {
      await runBrowserTest(workspace, take);
    } else {
      await runElectronAppTest(workspace, take);
    }
```